example/ipsec_crypto/odp_ipsec.c

```diff
@@ -634,7 +634,6 @@ pkt_disposition_e do_ipsec_in_classify(odp_packet_t *pkt,
 	/* Initialize parameters block */
 	memset(&params, 0, sizeof(params));
 	params.session = entry->state.session;
-	out_pkt = entry->in_place ? *pkt : ODP_PACKET_INVALID;
```

Comment:
Copyright can be updated.

Reply:
I'll do that later in a PR that will have more substantial changes to the code.